src/base58.cpp

```diff
@@ -4,16 +4,8 @@
 
 #include "base58.h"
 
-#include "hash.h"
-#include "uint256.h"
 
-#include <assert.h>
-#include <stdint.h>
 #include <string.h>
-#include <vector>
```

Comment:
Yes, that's true! Please disregard this WIP PR (it shouldn't have been submitted in the first place - I should have found another way to trigger Travis CI testing), see the final PR "Remove includes in .cpp files for things the corresponding .h file already included" instead :-)

Sorry about taking up reviewer time with this WIP stuff.